docker-compose.yml

```diff
@@ -1,17 +1,24 @@
-version: '3.2'
+version: '3.3'
 services:
   db:
     environment:
       MYSQL_ROOT_PASSWORD: ai_integration
-    image: mariadb:10.2
+    image: mariadb:10.3
```

Comment:
in the future, unrelated changes shouldn't be a part of a PR. this would be a PR titled 'updates dependencies' or something like this